lib/components/common/StyledCard.tsx

```diff
@@ -5,11 +5,11 @@ import CardContent from '@mui/material/CardContent';
 import CardActionArea from '@mui/material/CardActionArea';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them